src/components/ebay-button/index.js

```diff
@@ -77,6 +78,7 @@ function getInitialState(input) {
 
     return {
         htmlAttributes: processHtmlAttributes(input),
+        id: input.id || getWidgetId(out),
```

Comment:
Are there any possibility of binding issues? Does the custom ID get used as the Marko v3 element's ID through the lifecycle? Or, does it matter?

Reply:
It's not possible to access the id in any pre render lifecycle method so this won't cause any issues. There is one case that I didn't account for in my hacky utility that I am fixing now which is why I put this up as a draft.

After the CI is passing this should be okay.

Reply:
I pulled the branch and it doesn't seem to be working for me. I simply did this:
```
<ebay-button priority="primary" id="button-test">text test</ebay-button>
```

Reply:
Never mind. I was looking at the wrong element in my inspector!